### PR TITLE
VBADocuments: bulk status/report operation

### PR DESCRIPTION
**Pull request by patrickvinograd**
Will fix https://github.com/department-of-veterans-affairs/vets-contrib/issues/135 in concert with some additional API gateway configuration to establish rate limits. 

This filters out any status updates that don't need to pass through, and also aggregates the status calls that pass through to CMAPI into one operation (which likewise accepts up to 100 ids at a time). Combined with a reasonable rate limit specific to this operation I expect it to not generate too much load. But, I plan to test this against the dev environment to be sure before publicizing this operation.

### CODE REVIEW

File: modules/vba_documents/README.yml [hunk 1]
```diff
@@ -292,50 +325,85 @@ components:
...
+        - attributes
         - guid
         - status
         - code
         - message
         - detail
       properties:
-        guid:
-          description: The document upload identifier.
+        id:
```
Comment (jkassemi) [inline modules/vba_documents/README.yml]: minor irritation about inconsistent use of `id` vs `guid`

File: modules/vba_documents/app/controllers/vba_documents/v0/reports_controller.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+      MAX_REPORT_SIZE = 100
+
+      def create
+        statuses = VBADocuments::UploadSubmission.refresh_and_get_statuses!(params['guids'])
+        render json: statuses,
+               each_serializer: VBADocuments::UploadSerializer
+      end
+
+      def validate_params
```
Comment (jkassemi) [inline modules/vba_documents/app/controllers/vba_documents/v0/reports_controller.rb]: `private`

Comment (patrickvinograd) [reply modules/vba_documents/app/controllers/vba_documents/v0/reports_controller.rb]: Done

File: modules/vba_documents/README.yml (hunk 1 above)
Comment (patrickvinograd) [reply modules/vba_documents/README.yml]: Yeah...I should have thought this through better, with JSON API there's no need to have guid represented in the object attributes proper :( But I'm guessing it's being consumed at this point, so don't want to remove it. I could change the bulk report operation request object key to `ids` with a view towards deprecating the guids attribute and standardizing on 'ids'.

**Pull request merged**